app/test_main.py

```diff
@@ -1,3 +1,37 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_human_age_zero():
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_get_human_age_below_15():
+    assert get_human_age(14, 14) == [0, 0]
```

Comment:
The expected output for `get_human_age(14, 14)` might be incorrect depending on the logic of `get_human_age`. Ensure that the function correctly handles ages below 15.